aea/runtime.py

```diff
@@ -270,14 +278,26 @@ async def stop_runtime(self) -> None:
             await self.main_loop.wait_completed()
         self._teardown()
 
+        if self._storage is not None:
+            self._storage.stop()
+            await self._storage.wait_completed()
+
         self.multiplexer.stop()
         await self.multiplexer.wait_completed()
         self.logger.debug("Runtime loop stopped!")
 
     async def run_runtime(self) -> None:
         """Run agent and starts multiplexer."""
         self._state.set(RuntimeStates.starting)
-        await asyncio.gather(self._start_multiplexer(), self._start_agent_loop())
+        await asyncio.gather(
+            self._start_multiplexer(), self._start_agent_loop(), self._start_storage()
+        )
+
+    async def _start_storage(self) -> None:
+        """Start storage component."""
+        self._storage = Storage("sqlite://:memory:", threaded=True)
```

Comment:
No magic values; needs to be configurable

Reply:
done